src/replica/ConfigTable.h

```diff
@@ -0,0 +1,173 @@
+/*
+ * LSST Data Management System
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+#ifndef LSST_QSERV_REPLICA_CONFIGTABLE_H
+#define LSST_QSERV_REPLICA_CONFIGTABLE_H
+
+// System headers
+#include <cstdint>
+#include <list>
+#include <iosfwd>
+#include <string>
+#include <vector>
+
+// Third party headers
+#include "nlohmann/json.hpp"
+
+// Qserv headers
+#include "replica/Common.h"
+
+// This header declarations
+namespace lsst::qserv::replica {
+
+/**
+ * Class TableInfo encapsulates various parameters describing tables.
+ * @note The interface is not thread-safe.
+ */
+class TableInfo {
+public:
+    std::string name;      ///< The name of the table.
+    std::string database;  ///< The name of the parent database.
+
+    // The publishing status and major events in the table's history.
+
+    bool isPublished = false;
+    uint64_t createTime = 0;
+    uint64_t publishTime = 0;
```

Comment:
This suggestion has been registered at my TODO list: https://confluence.lsstcorp.org/display/DM/Completed%2C+on-going+and+proposed+improvements.
I will do this in the next refactoring as there are many uses of this plain integer type instead of `<crono>`.